Cannot find module 'dox'

When i execute `npm run docs`, i have this error : 

```console
C:\markdown-magic>npm run docs

> markdown-magic@0.1.6 docs C:\markdown-magic
> node examples/generate-readme.js

original innerContent This will replace all the contents of inside the comment DUDE
{ optionOne: 'DUDE' }
module.js:472
    throw err;
    ^

Error: Cannot find module 'dox'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.RENDERDOCS (C:\markdown-magic\examples\generate-readme.js:17:24)
    at updateContents (C:\markdown-magic\lib\updateContents.js:25:50)
    at transformsToRun.forEach (C:\markdown-magic\lib\processFile.js:81:26)
    at Array.forEach (native)
    at processFile (C:\markdown-magic\lib\processFile.js:79:21)
    at files.forEach (C:\markdown-magic\index.js:29:20)

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\forresst\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "docs"
npm ERR! node v7.3.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! markdown-magic@0.1.6 docs: `node examples/generate-readme.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the markdown-magic@0.1.6 docs script 'node examples/generate-readme.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the markdown-magic package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node examples/generate-readme.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs markdown-magic
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls markdown-magic
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\markdown-magic\npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
   },
   "devDependencies": {
     "ava": "^0.17.0",
+    "dox": "^0.9.0",
     "eslint": "^3.11.1",
     "eslint-config-airbnb-base": "^10.0.1",
     "eslint-plugin-import": "^2.2.0",

